docs/_posts/2018-01-14-42.2.0-release.md

```diff
@@ -0,0 +1,205 @@
+---
+title: PostgreSQL JDBC Driver 42.2.0 Released
+date: 2018-01-14 15:50:46 -0500
+categories:
+  - new_release
+version: 42.2.0
+---
+**Notable changes**
+
+
+## [42.2.0] (2017-12-21)
```

Comment:
Please remove this line from `_posts/2018-01-14-42.2.0-release.md`. Version title is generated automatically by Jekyll (see https://github.com/pgjdbc/pgjdbc/blame/5d585aac7e4f916d4b54dccd11c778143a1f7725/docs/documentation/changelog.md#L80 )